docs/sdk-and-tools/sdk-py/mxpy-cli.md

```diff
@@ -530,35 +530,58 @@ If your address is guarded by another wallet, you'll still need to provide the `
 
 [comment]: # (mx-context-auto)
 
-## Relayed V3 transactions
+## Relayed transactions V3
 
 Relayed transactions are transactions with the fee paid by a so-called relayer. In other words, if a relayer is willing to pay for a transaction, it is not mandatory for the sender to have any EGLD for fees. To learn more about relayed transactions check out [this page](/developers/relayed-transactions/).
 
-In this section we'll see how we can send `Relayed V3` transactions using `mxpy`. For a more detailed look on `Relayed V3` transactions, take a look [here](/developers/relayed-transactions/#relayed-transactions-version-3). For these kind of transactions a new transaction field has been introduced, called `innerTransactions`. In this example we'll see how we can create both the inner transactions and the relayed transaction.
+In this section we'll see how we can send `Relayed V3` transactions using `mxpy`. For a more detailed look on `Relayed V3` transactions, take a look [here](/developers/relayed-transactions/#relayed-transactions-version-3). For these kind of transactions two new transaction fields were introduced, `relayer` and `relayerSignature`. In this example we'll see how we can create the relayed transaction.
 
-### Creating the inner transactions
+For this, a new command `mxpy tx relay` has been added. The command can be used to relay a previously signed transaction. The saved transaction can be loaded from a file using the `--infile` argument.
 
-We can simply create the inner transactions the same way we did above, by using the `mxpy tx new` command. The only difference is that we'll have to provide an additional argument called `--inner-transactions-outfile`, which represents the file where the inner transactions are saved to be later used by the relayer. To keep it simple, we'll send 1 EGLD from Alice to Bob, and Carol will be the relayer. To create the EGLD transfer transaction from Alice to Bob, we run the following command:
+There are two options when creating the relayed transaction:
+1. Create the inner transaction separately. (Sender signature and relayer signature are added in different steps.)
```

Comment:
maybe 

1. Create the relayed transaction
2. Add the signatures (sender and relayer)

Reply:
Now it is only one transaction so the differentiation between an inner transaction and a relayed transaction seems unnecessary